packages/contracts/contracts/voting/majority/MajorityVotingBase.sol

```diff
@@ -149,18 +136,11 @@ abstract contract MajorityVotingBase is
     }
 
     /// @inheritdoc IMajorityVoting
-    function changeVoteSettings(
-        uint64 _supportThreshold,
-        uint64 _minParticipation,
-        uint64 _minDuration,
-        uint256 _minProposerVotingPower
-    ) external auth(CHANGE_VOTE_SETTINGS_PERMISSION_ID) {
-        _validateAndSetSettings(
-            _supportThreshold,
-            _minParticipation,
-            _minDuration,
-            _minProposerVotingPower
-        );
+    function updateMajorityVotingSettings(MajorityVotingSettings calldata _majorityVotingSettings)
```

Comment:
From the SDK side we call it `setXXX`, although to the plugin, not the abstract contract

Reply:
I tried to have matching function and event names.

`function updateVoteSettings` and`emit VoteSettingsUpdated`
instead of
`function setVoteSettings` and`emit VoteSettingsSet`

If you prefer the latter, I can change it.